sklearn/manifold/spectral_embedding_.py

```diff
@@ -44,6 +44,8 @@ def _graph_connected_component(graph, node_id):
         node
     """
     n_node = graph.shape[0]
+    graph = sparse.csc_matrix(graph)
```

Comment:
This does not look right -- it is converted to CSR on the following line.

Reply:
Thanks, I think you are right.

To be compliant with jnothman's suggestion on May 8th about converting and storing it as a graph at the top, I suggest the following actions:

Change line 48 to: `graph = sparse.csr_matrix(graph)`
Delete lines 49-51

Reply:
I'm a bit confused about this. @jnothman?

Reply:
> To be compliant with jnothman's suggestion on May 8th about converting and storing it as a graph at the top, I suggest the following actions:

What I said was "Can we just convert and store as graph at the top of the function?"

What I meant was "Can we just convert the matrix to an array and assign it back to the `graph variable at the top of the function?"

Reply:
> This does not look right -- it is converted to CSR on the following line.

Actually, since it must be a symmetric matrix, its row and column matrix representations are equivalent. Thus it was passing the tests.